src/Microsoft.NetCore.Analyzers/UnitTests/Security/DoNotDisableSchUseStrongCryptoTests.cs

```diff
@@ -152,7 +151,8 @@ public void TestMethod()
         string switchName = ""Switch.System.Net.DontEnableSchUseStrongCrypto"";
         AppContext.SetSwitch(switchName, true);
     }
-}");
+}",
+            GetCSharpResultAt(9, 9, DoNotSetSwitch.DoNotDisableSchUseStrongCryptoRule, "SetSwitch"));
```

Comment:
Unrelated to your PR, but we should also write a generic code quality analyzer that flags non-const local variables that are always assigned constant values and suggest to convert it to a const field or local variable.